tests_zemu/package.json

```diff
@@ -19,38 +19,40 @@
     "format:check": "FORCE_COLOR=1 prettier --check .",
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
-    "test": "jest",
+    "test": "yarn clean && jest --maxConcurrency 3",
```

Comment:
Any reason why we just run this into 3?

Reply:
we made this in other apps to avoid some sporadic failures that might appear in Zemu tests